Core/Core/People/PeopleListViewController.swift

```diff
@@ -172,7 +172,14 @@ extension PeopleListViewController: UISearchBarDelegate {
     }
 
     public func searchBarSearchButtonClicked(_ searchBar: UISearchBar) {
-        searchBar.searchTextField.resignFirstResponder()
+        if #available(iOSApplicationExtension 13.0, *) {
+            searchBar.searchTextField.resignFirstResponder()
+        } else if let searchField = searchBar.value(forKey: "_searchField") as? UITextField {
+            searchField.resignFirstResponder()
+        } else {
+            view.endEditing(true)   //  TODO: - TEST THIS!!
+        }
```

Comment:
Why was this change needed? Is the `searchField` not available sometimes? Why didn't it break before?

Reply:
yeah, agreed, this one is weird, xcode complaining about it